tests/test_app/resources.py

```diff
@@ -0,0 +1,54 @@
+from .models import Bot, User
```

Comment:
Your library is a plugin for SQLAlchemy, so I wouldn't assume it is used with flask or even a web framework. Perhaps someone wants to use it with some simple script? So let's just leave db models and write test using them.